Jenkinsfile

```diff
@@ -28,4 +28,8 @@ fhBuildNode {
             name = 'fh-dataman'
         }
     }
+
+    stage('Build Image') {
+        dockerBuildNodeComponent("fh-dataman")
```

Comment:
@sedroche here, either add a second param called "rhmap" for the dockerhub org, or move the image repo on dockerhub to the feedhenry org. I prefer the second option, since this is an open repo in the feedhenry github org.